py3dtiles/gltf_material.py

```diff
@@ -3,12 +3,13 @@
 
 
 class GlTFMaterial():
-    def __init__(self, metallicFactor=0, roughnessFactor=0, rgb=[1, 1, 1], alpha=1):
+    def __init__(self, metallicFactor=0, roughnessFactor=0, rgb=[1, 1, 1], alpha=1, textureUri=None):
```

Comment:
Isn'it best to initialize RGB values to 0, so that object are white by default ? 
If we get an object with a texture only, it will not be visible I think ?